Jasmine/Tetris/ViewModels/TetrisGameViewModel.swift

```diff
@@ -1,4 +1,200 @@
 import Foundation
 
+/// The main view model for Tetris game
 class TetrisGameViewModel {
```

Comment:
Make this class extend `CountdownTimable`, in this class or otherwise. (extensions?)